ibis/backends/base/sql/alchemy/__init__.py

```diff
@@ -335,26 +333,53 @@ def current_database(self) -> str:
         return self.database_name
 
     @util.deprecated(version='2.0', instead='`list_databases`')
-    def list_schemas(self):
+    def list_schemas(self, like: str | None = None) -> list[str]:
         return self.list_databases()
 
     def _log(self, sql):
         try:
             query_str = str(sql)
-        except sqlalchemy.exc.UnsupportedCompilationError:
+        except sa.exc.UnsupportedCompilationError:
             pass
         else:
             util.log(query_str)
 
-    def _get_sqla_table(self, name, schema=None, autoload=True):
-        return sqlalchemy.Table(
-            name, self.meta, schema=schema, autoload=autoload
-        )
-
-    def _sqla_table_to_expr(self, table):
-        node = self.table_class(table, self)
+    def _get_sqla_table(
+        self,
+        name: str,
+        schema: str | None = None,
+        autoload: bool = True,
+        **kwargs: Any,
+    ) -> sa.Table:
+        return sa.Table(name, self.meta, schema=schema, autoload=autoload)
+
+    def _sqla_table_to_expr(self, table: sa.Table) -> ir.TableExpr:
+        schema = self._schemas.get(table.name)
+        node = self.table_class(table, self, schema)
         return self.table_expr_class(node)
 
+    def table(
+        self,
+        name: str,
+        **kwargs: Any,
+    ) -> ir.TableExpr:
+        """Create a table expression from a table in the SQLite database.
```

Comment:
I assume SQLite shouldn't be mentioned here.

Reply:
Yes, thank you :)

Reply:
Done.